src/rtextures.c

```diff
@@ -3730,9 +3730,9 @@ void ImageDrawTriangleEx(Image *dst, Vector2 v1, Vector2 v2, Vector2 v3, Color c
     }
 
     // Calculate the initial barycentric coordinates for the top-left point of the bounding box
-    int w1Row = (xMin - v2.x)*w1XStep + w1YStep*(yMin - v2.y);
-    int w2Row = (xMin - v3.x)*w2XStep + w2YStep*(yMin - v3.y);
-    int w3Row = (xMin - v1.x)*w3XStep + w3YStep*(yMin - v1.y);
+    int w1Row = (int)((xMin - v2.x) * w1XStep + w1YStep * (yMin - v2.y));
```

Comment:
Please, could you review formating, in raylib I always use the `*` and `/` operators without spaces around.

Reply:
Fixed